src/main/java/com/openlattice/data/DataApi.java

```diff
@@ -146,6 +145,15 @@ Void storeAssociationData(
     @PATCH( BASE + "/" + ENTITY_DATA )
     Void createEntityAndAssociationData( @Body BulkDataCreation data );
 
+    @POST( BASE + "/" + ENTITY_KEY_ID )
+    UUID getEntityKeyId( @Body EntityKey entityKey );
+
+    @POST( BASE + "/" + ENTITY_KEY_ID )
+    Map<EntityKey, UUID> getEntityKeyIds( @Body Set<EntityKey> entityKeys );
```

Comment:
Did you build a custom serializer / test this? You can't have objects be the keys of maps.